REFERENCE.md

```diff
@@ -810,6 +810,14 @@ Reuse ssl sessions
 
 Default value: `true`
 
+##### `ssl_client_renegotiation`
+
+Data type: `Boolean`
+
+Allow ssl client renegotiation
+
+Default value: `false`
```

Comment:
note: this is Ok to add, but I'd try to generate it with `rake strings:reference` if you didn't already.